apps/system/utils/Kconfig

```diff
@@ -12,7 +12,7 @@ config KERNEL_CMDS
 config FS_CMDS
 	bool "FS shell commands"
 	default y
-	depends on TASH && (NFILE_DESCRIPTORS != 0)
+	depends on !BUILD_PROTECTED && TASH && (NFILE_DESCRIPTORS != 0)
```

Comment:
Do we need to block all of fs cmds with build_protected?

Reply:
I will modify

not depend BUILD_POTECTED : cat, cd, echo, ls, mkdir, pwd, rm, rmdir, 
depend BUILD_POTECTED : mkrd, mksmartfs, mount, umount, df